source/utils.ts

```diff
@@ -4,8 +4,16 @@
  */
 
 import { ESLintUtils } from "@typescript-eslint/experimental-utils";
+import { TSESTree as es } from "@typescript-eslint/experimental-utils";
```

Comment:
question: why two import statements from the same package?

Reply:
Good catch!